opentracing/src/main/java/opentracing/Span.java

```diff
@@ -25,16 +25,7 @@
   /**
    * Suitable for serializing over the wire, etc.
    */
-  TraceContext traceContext();
-
-  /**
-   * Denotes the beginning of a subordinate unit of work.
-   *
-   * @param operationName name of the operation represened by the new span from the perspective of
-   * the current service.
-   * @return a new child Span in "started" state.
-   */
-  Span startChild(String operationName);
+  SpanContext childContext();
```

Comment:
Some libraries divorce the concern of creating a span vs adding to it. Ex. this the case in Google Cloud Trace, Sleuth and Brave: you have a reference to add to a span another object created for you.

When trying to implement OT in spring-cloud-sleuth, was that adding a branch ability to the span itself (vs the tracer), requires some relatively heavy changes, such as needing to hold references to what's currently separate instrumentation code. It makes span more a "scopable tracer" and less a DTO that has helpers around time.